testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkCompatibility.java

```diff
@@ -1427,6 +1428,12 @@ public void testTrinoReadsSparkRowLevelDeletes(StorageFormat tableStorageFormat,
         assertThat(onTrino().executeQuery("SELECT * FROM " + trinoTableName)).containsOnly(expected);
         assertThat(onSpark().executeQuery("SELECT * FROM " + sparkTableName)).containsOnly(expected);
 
+        // Delete to a file that already has deleted rows
```

Comment:
is this exercised by a non-product-test as well?

Reply:
`BaseConnectorTest` cannot guarantee that because it doesn't know about "files".
Unless it deletes "a lot".